src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,46 @@
 package core.basesyntax.service;
 
+import core.basesyntax.InvalidUserDataException;
 import core.basesyntax.dao.StorageDao;
-import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    private static final int MIN_LENGTH = 6;
+    private static final int MIN_AGE = 18;
+    private StorageDao storageDao;
```

Comment:
Consider making the `storageDao` field final if it is not expected to change after initialization. This ensures the field is safely published in a multi-threaded environment and makes the class more robust.